packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -190,6 +190,7 @@ export class Chip {
     return (
       <Host
         class={{
+          dismissed: !visible,
```

Comment:
```suggestion
          'ic-chip-dismissed': !visible,
```
Please can you make the class name more unique\component specific as there have been issues in the past with classNames like "icon", "disabled" etc on the host element where styles leak into the components eg if someone had a ".dismissed" selector in their app css